src/test/ui/conditional-compilation/cfg-attr-multi-false.rs

```diff
@@ -4,7 +4,7 @@
 // compile-pass
 
 #![warn(unused_must_use)]
-#![feature(cfg_attr_multi)]
+#![cfg_attr(stage0, feature(cfg_attr_multi))]
```

Comment:
`cfg_attr(stage0, ...)` is not required in tests, the features can just be removed.